processing/src/main/java/org/apache/druid/segment/DimensionHandler.java

```diff
@@ -143,7 +143,7 @@ DimensionMergerV9 makeMerger(
    */
   Comparator<ColumnValueSelector> getEncodedValueSelectorComparator();
 
-  /**
+  /**q
```

Comment:
Q: q?

:trollface: qq

Reply:
lol oops, i suspect this is an artifact of stupid keyboard lag when switching 'spaces' to the one with my terminal on mac